client/src/components/ChatPage/ChatPage.tsx

```diff
@@ -22,14 +34,15 @@ export function ChatPage() {
         <ChatList
           onCreateNewClick={onCreateNewClick}
           isNewChatPending={isNewChatPending}
+          onChatClick={onChatClick}
         />
       </section>
       <section className={styles['chat-section']}>
-        {isNewChatPending ? (
+        {!selectedConversation && !isNewChatPending && <WelcomeChatWindow />}
```

Comment:
this should be extracted in a useMemo because it looks bloated here
```
const render = useMemo(() => {
 if (isNewChatPending) return <New chat />;
 if (selectedConversation) return <Chat />;
 return <WelcomeChatWindow />;
}, [deps])
```
and then here simply have `{render}`